tensorflow/python/autograph/operators/py_builtins.py

```diff
@@ -322,10 +334,30 @@ def print_(*objects, **kwargs):
   else:
     _py_print(*objects, **kwargs)
 
-
 def _py_print(*objects, **kwargs):
   print(*objects, **kwargs)
 
+def max_(*args, **kwargs):
+  if any(tensor_util.is_tf_type(s) for s in args):
+    return _tf_max(*args, **kwargs)
+  return _py_max(*args, **kwargs)
+
+def _tf_max(*args, **kwargs):
+  if len(kwargs):
+    kwargs_tuple = tuple(set(kwargs.keys()))
+    raise ValueError('These keyword arguments are ' 
+                     'currently not supported: {}'.format(kwargs_tuple))
+  elif len(args) == 1:
+    # TODO (bhack) Do we want the input Tensor to be rank==1 or scalar?
+    # if not something like [[1, 2][3,4]] it will give an unexpected result
+    return  math_ops.reduce_max(*args, axis=0)
+  else:
```

Comment:
Nit: else is unnecessary since the other branches either raise or return. Linear code is easier to read.